fdbserver/workloads/RemoveServersSafely.actor.cpp

```diff
@@ -386,7 +388,7 @@ struct RemoveServersSafelyWorkload : TestWorkload {
 		return killProcArray;
 	}
 
-	ACTOR static Future<Void> removeAndKill( RemoveServersSafelyWorkload* self, Database cx, std::set<AddressExclusion> toKill, std::set<AddressExclusion>* pIncAddrs)
+	ACTOR static Future<Void> removeAndKill( RemoveServersSafelyWorkload* self, Database cx, std::set<AddressExclusion> toKill, std::set<AddressExclusion>* pIncAddrs, bool safeKillSet)
```

Comment:
Parameter safeKillSet is not very intuitive. 
Suggestion: Change it to `markExcludeAsFailed`